### PR TITLE
fix minor typos

### PR DESCRIPTION
**Pull request by MichaelBitard**
While reading this, I fixed some small stuff I saw.

### CODE REVIEW

File: docs/piaf/reader/report_january.md [hunk 1]
```diff
@@ -3,19 +3,26 @@
 ## Context
 At the dawn of launching a new project on the CRPA data to further test the opportunity of using PIAF in the context of the French administration, we were facing concerns regarding the standardization of the data we were to feed to PIAF.
 
-We want to be able to launch a new instance of PIAF with as few human intervention as possible. However the performance of the whole pipeline is much influenced by the size of the knowledge base available for searching. We also know that many parameters will influence the answer retrieval in terms of quality (number of correct retrievals) and response-time. These parameters can be : document length, number of documents retreived, number of answers requested per documents, among others.
+We want to be able to launch a new instance of PIAF with as few human interventions as possible.
```
Comment (psorianom) [inline docs/piaf/reader/report_january.md:6]: I don't think that we want bullet points here. I believe that these paragraphs are not an enumeration of events/ideas but 
single logical sequences.

**Review by psorianom — CHANGES_REQUESTED** [review]
Hi @MichaelBitard . Thanks for your corrections. I just have a comment regarding the paragraph-to-list changes you made. I think they should be kept as paragraphs.

File: docs/piaf/reader/report_january.md (hunk 1 above)
Comment (MichaelBitard) [reply docs/piaf/reader/report_january.md:6]: I did not want to create bullet points and I think the output will be exactly the same. I understand the idea of logical sequence, it's just that having some newlines in the .md simplifies the document's readability/git diffs (IMHO).

I can revert that if you prefer having only one line in the markdown

Comment (psorianom) [reply docs/piaf/reader/report_january.md:6]: oh I see! I did not check the output. Having single lines seems better indeed :+1:

**Review by psorianom — APPROVED** [review]

**Pull request merged**